extensions/amp-consent/0.1/consent-info.js

```diff
@@ -328,27 +367,35 @@ export function getConsentStateValue(enumState) {
 }
 
 /**
- * Converts metadata to stroage value:
- * {'gdprApplies': true, 'consentStringType': 'tcf-v2'} =>
- * {'ga': true, 'cst': 'tcf-v2'}
+ * Converts ConsentMetadataDef to stroage value:
+ * {'gdprApplies': true, 'consentStringType': 2} =>
+ * {'ga': true, 'cst': 2}
  *
- * @param {Object} unused
+ * @param {ConsentMetadataDef=} consentInfoMetadata
  * @return {Object}
  */
-function composeMetadataStoreValue(unused) {
+export function composeMetadataStoreValue(consentInfoMetadata) {
   const storageMetadata = map();
-  // TODO(micajuineho) if (metadata['gdprApplies']) {...}
+  if (consentInfoMetadata['consentStringType']) {
```

Comment:
Is it possible to use a for loop to go through all the `consentInfoMetadata` fields?

Reply:
Not unless we want to keep a mapping of `ConsentMetadataDef` => `METADATA_STORAGE_KEY`. Also, for `#composeStoreValue` we manually look and set each field (because different work is done for each field), and that might be the case for this storage object as well.

Reply:
OK we can decide when you add additional field.